config/config.py

```
@@ -1,292 +1,471 @@
-RESTART_REQUIRED_KEYS = ["TAUTULLI_API_KEY", "TAUTULLI_URL", "DISCORD_TOKEN"]
+                finally:
+                    # Release lock
+                    fcntl.flock(config_file.fileno(), fcntl.LOCK_UN)
+                    
+    except Exception as e:
+        # Clean up temporary file if it exists
+        if temp_file_path and os.path.exists(temp_file_path):
+            try:
+                os.unlink(temp_file_path)
+            except OSError:
+                pass  # Best effort cleanup
```

Ruff

Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

(SIM105)